src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -10,6 +10,7 @@ import { TextStyleType } from "./utils/getTextStyles";
 import { EMPTY_VALUE, parseRawValue } from "./utils/parseRawValue";
 
 export type RichTextEditorProps = {
+    id?: string;
```

Comment:
We can use the Id generator as a fallback

Reply:
Added